src/emm/emm_cli.py

```diff
@@ -211,5 +222,22 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100))
+@click.argument("revision", nargs=-1)
```

Comment:
I tend to prefer options over arguments. This also needs better documentation and error handling.